scripts/qa_crowbarsetup.sh

```diff
@@ -1863,7 +1867,7 @@ function onadmin_allocate
     else
         if [[ $hacloud = 1 ]] ; then
             cluster_node_assignment
-            local nodes=($unclustered_nodes)
+            local nodes=(${unclustered_nodes[@]})
```

Comment:
That doesn't work:
~~~
$ a=(1 2 3)
$ declare -p a
declare -a a='([0]="1" [1]="2" [2]="3")'
$ b=$a
$ declare -p b
declare -- b="1"
$ b=($a)
$ declare -p b
declare -a b='([0]="1")'
~~~